lib/Agrammon/Web/Routes.pm6

```diff
@@ -500,6 +502,26 @@ sub application-routes(Agrammon::Web::Service $ws) {
             }
         }
 
+        post -> LoggedIn $user, 'upload' {
+            request-body -> (:$file, :datasetName($dataset-name), :$comment) {
```

Comment:
Probably should mark some of these required

Reply:
Marked the first two as required, `$comment` is optional.